modules/graph/fragment/arrow_fragment_builder_impl.h

```diff
@@ -996,32 +1000,64 @@ ArrowFragment<OID_T, VID_T, VERTEX_MAP_T, COMPACT>::AddEdgesToExistedLabel(
     for (auto v : iv) {
       auto offset = vertex_offset(v);
       vid_t src_id = vid_parser_.GenerateId(v_label, offset);
-      // if(!directed_ && vis.count(src_id)) continue;
       auto oe = this->GetOutgoingAdjList(v, label_id);
-      // if(oe.Size()) prev_src_ids.push_back(src_id);
       for (auto& e : oe) {
         auto dst_label = vertex_label(e.neighbor());
         auto dst_offset = vertex_offset(e.neighbor());
         vid_t dst_id = vid_parser_.GenerateId(dst_label, dst_offset);
```

Comment:
Declare "std::to_string(src_id) + std::to_string(dst_id)" and "std::to_string(dst_id) + std::to_string(src_id)" at the beginning of the loop, to avoid concat them again and again, also improves readability